py_ballisticcalc/interface.py

```diff
@@ -1,9 +1,8 @@
 """Implements basic interface for the ballistics calculator"""
 from dataclasses import dataclass, field
 
-from .conditions import Atmo, Shot
-from .munition import Weapon, Ammo
-# pylint: disable=import-error,no-name-in-module
+from .conditions import Shot
+# pylint: disable=import-error,no-name-in-module,wildcard-import,unused-wildcard-import
 from .backend import *
```

Comment:
maybe we need to define backend's __all__ property and remove the wildcard import

Reply:
How is this done?

Reply:
Skip it now, I'll do it by myself after merge